app/src/main/res/layout-sw600dp/pregamepage.xml

```diff
@@ -0,0 +1,131 @@
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/autoPageLayout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+
+        <TableLayout
+            android:id="@+id/pregameTableLayout"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_alignParentStart="true"
+            android:layout_alignParentLeft="true"
+            android:layout_alignParentTop="true">
+
+            <TableRow
+                android:id="@+id/startPosition"
+                android:layout_width="fill_parent"
+                android:layout_marginBottom="17dp"
+                android:weightSum="4">
+
+                <TextView
+                    android:id="@+id/pregame0"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_alignTop="@+id/tableLayout"
+                    android:layout_alignParentEnd="true"
+                    android:layout_alignParentRight="true"
+                    android:layout_weight="1"
+                    android:gravity="center_vertical"
+                    android:text="Start Position: "
+                    android:textSize="@dimen/text_medium" />
+
+                <RadioGroup
+                    android:id="@+id/startPositionGroup"
+                    android:layout_width="fill_parent"
+                    android:layout_height="match_parent"
+                    android:layout_weight="3"
+                    android:gravity="center"
+                    android:orientation="horizontal">
+
+                    <RadioButton
+                        android:id="@+id/hab1"
+                        android:layout_width="wrap_content"
+                        android:layout_height="match_parent"
+                        android:layout_alignTop="@+id/tableLayout"
+                        android:layout_alignParentStart="true"
+                        android:layout_alignParentLeft="true"
+                        android:layout_weight="1"
+                        android:checked="false"
+                        android:text="HAB 1"
+                        android:textSize="@dimen/text_small" />
+
+                    <RadioButton
+                        android:id="@+id/hab2"
+                        android:layout_width="wrap_content"
+                        android:layout_height="match_parent"
+                        android:layout_alignTop="@+id/tableLayout"
+                        android:layout_alignParentStart="true"
+                        android:layout_alignParentLeft="true"
+                        android:layout_weight="1"
+                        android:text="HAB 2"
+                        android:textSize="@dimen/text_small" />
```

Comment:
Maybe "Level 1" and "Level 2".

I think we should record if it starts on the left, middle or right in a spinner along with which level.